.github/workflows/pr-build-live-branch.yml

```
@@ -0,0 +1,87 @@
+            - name: Checkout repository
+              uses: actions/checkout@v4
+
+            - name: Get current version
+              id: version
+              run: |
+                  current_version=$( grep -oP '(?<=Version: )(.+)' woocommerce-gateway-stripe.php | head -n1 )
+                  echo "Detected version: $current_version"
+                  echo "version=$current_version" >> $GITHUB_OUTPUT
+
+            - name: Setup PHP and Node
+              uses: ./.github/actions/setup-env
```

actionlint (1.7.11)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/setup-env": line 5: unexpected key "type" for definition of input "php-version"

(action)